source/dynk.f90

```diff
@@ -2503,7 +2556,36 @@ real(kind=fPrec) function dynk_getvalue(element_name, att_name)
           goto 100 ! ERROR
         end if
 
-        ! Not yet supported : Multipoles (11)
+
+      case(11)
+      im=irm(ii)
```

Comment:
Please follow the indentation of the other cases. The content of a case should be indented one level.